content/guides/azure-todo-static-web-app.md

```diff
@@ -0,0 +1,747 @@
+---
+title: Building Azure Static Web Apps with Neon
+subtitle: A step-by-step guide to creating and deploying static sites using Azure and Neon Postgres
+author: dhanush-reddy
+enableTableOfContents: true
+createdAt: '2024-12-14T00:00:00.000Z'
+updatedOn: '2024-12-14T00:00:00.000Z'
+---
+
+If you've been looking for a modern approach to deploying web applications without managing traditional server infrastructure, [Azure Static Web Apps](https://azure.microsoft.com/en-us/products/app-service/static) might be exactly what you need. At its core, it's a service optimized for hosting static assets with global distribution, but its true power lies in the seamless integration with [Azure Functions](https://azure.microsoft.com/en-us/products/functions) for backend operations.
```

Comment:
how about making the first person or second person consistent throughout the blog?

Reply:
Please check now, have made it consistent in second person